<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="Receive_window_TextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>431</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="transmit_window_TextEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Send_Btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Transceive_GroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Transceive</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Setting_GroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="Remote_Port_Label">
       <property name="text">
        <string>Remote Port:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Remote_Port_LineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Local_Port_Label">
       <property name="text">
        <string>Local Port:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Local_Port_lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>192</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="Protocol_Label">
       <property name="font">
        <font>
         <family>Ubuntu</family>
        </font>
       </property>
       <property name="text">
        <string>Protocol:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="Protocol_ComboBox">
       <item>
        <property name="text">
         <string>TCP Server</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TCP_Client</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UDP</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="Remote_IP_Label">
       <property name="text">
        <string>Remote host IP:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLineEdit" name="Remote_IP_LineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Status_Label">
       <property name="text">
        <string>status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Connect_Btn">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>194</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QCheckBox" name="HEX_Display_CheckBox">
       <property name="text">
        <string>Show Hex</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="HEX_Transmit_CheckBox">
       <property name="text">
        <string>Send Hex</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Clear_Display_PushButton">
       <property name="text">
        <string>Clear Display</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Clear_Transmit_PushButton">
       <property name="text">
        <string>Clear Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>122</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="Auto_Transmit_CheckBox">
       <property name="text">
        <string>Auto transmit</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
